boards/arm/nrf51_blenano/doc/nrf51_blenano.rst

```diff
@@ -0,0 +1,109 @@
+.. _nrf51_blenano:
+
+Redbear Labs Nano
+#################
+
+Overview
+********
+
+The Nano is a development board equipped with Nordic's nRF51822 Bluetooth Low Energy SOC.
+This board is available on `RedBear Store`_.
+
+Hardware
+********
+
+nRF51 BLE Nano has two external oscillators. The frequency of the slow clock
+is 32.768 kHz. The frequency of the main clock is 16 MHz.
+
+Supported Features
+==================
+
+The nrf51_blenano board configuration supports the following nRF51
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| NVIC      | on-chip    | nested vectored      |
+|           |            | interrupt controller |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial port          |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth            |
++-----------+------------+----------------------+
+
+Connections and IOs
+====================
+
+BLE nano pinout
+
+.. image:: img/nano.png
+   :width: 500px
+   :align: center
+   :alt: BLE Nano
+
+DAPLink board
+
+.. image:: img/daplink.png
+   :width: 500px
+   :align: center
+   :alt: DAPLink
+
+The DAPLink USB board acts as a dongle. DAPLink debug probes appear on the host computer as a USB disk.
+It also regulates 5V from USB to 3.3V via the onboard LDO to power Nano.
+
+More information about Nano and DAPLink can be found at the `RedBear Github`_.
+
+Programming and Debugging
+*************************
+
+Applications for the ``nrf51_blenano`` board configuration can be built and
+flashed in the usual way (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Flashing
+========
+
+To flash an application, you'll need to connect your BLE Nano with the
+DAPLink board, then attach that to your computer via USB.
+
+.. warning::
+
+   Be careful to mount the BLE Nano correctly! The side of the board
+   with the VIN and GND pins should face **towards** the USB
+   connector.
```

Comment:
Maybe add...

     The `RedBear Store`_ page links to a tutorial video that shows how to
     properly solder headers and assemble the DAPLink and BLE Nano boards.

Reply:
Sure, updated as suggested.